packages/back-end/src/api/openapi/paths/putSavedGroup.yaml

```diff
@@ -0,0 +1,41 @@
+parameters:
+  - $ref: "../parameters.yaml#/id"
+tags:
+  - saved-groups
+summary: Update a single saved group
+operationId: putSavedGroup
+x-codeSamples:
+  - lang: "cURL"
+    source: |
+      curl -X PUT https://api.growthbook.io/api/v1/saved-groups/grp_123abc \
+        -H "Content-Type: application/json" \
+        -u secret_abc123DEF456: \
+        -d '{"groupName": "interal-users", "groupList": "userId-123, userId-345, userId-678"}'
+requestBody:
+  required: true
+  content:
+    application/json:
+      schema:
+        type: object
+        required: []
```

Comment:
It looks like they'd be allowed to post an empty object, should that be allowed? Which fields are required for this to function correctly?

Reply:
@tinahollygb You're correct. Technically, the only field that is required is the `id` field, otherwise, all other fields are optional - this gives the user the ability to update as much or as little as they would like. EX: They can update everything, or they can update any single field. 

If they do pass an empty object in, nothing will be updated.

Actually, the `dateUpdated` will still be updated. 

So, I can do a few different things - 1, I can make atleast one field required (maybe the `groupName`) OR, I can add a check to see if the body is empty, and if so, throw an error.

Any thoughts?